client/src/pages/course/team-distributions.tsx

```diff
@@ -0,0 +1,43 @@
+import { ProfileCourseDto } from 'api';
+import withSession from 'components/withSession';
+import { SessionProvider } from 'modules/Course/contexts';
+import { noAccessResponse, notAuthorizedResponse } from 'modules/Course/data';
+import { TeamDistributions } from 'modules/TeamDistribution/pages/TeamDistributions';
+import { GetServerSideProps } from 'next';
+import { CoursePageProps } from 'services/models';
+import { UserService } from 'services/user';
+import { getTokenFromContext } from 'utils/server';
+
+export const getServerSideProps: GetServerSideProps<{ course: ProfileCourseDto }> = async ctx => {
```

Comment:
You can use `getCourseProps` from client\src\modules\Course\data\getCourseProps.ts instead of creating `getServerSideProps`. Seems like functionality is the same

Reply:
done